logprep/util/json_handling.py

```diff
@@ -84,11 +85,14 @@ def parse_json(json_path):
 
     Returns
     -------
-    dict
-        The dictionary representing the json.
+    list
+        A list of dictionary representing the json/jsons.
     """
     with open(json_path, "r", encoding="utf8") as json_file:
-        return json.load(json_file)
+        parsed_json = json.load(json_file)
+        if isinstance(parsed_json, dict):
+            parsed_json = [parsed_json]
+        return parsed_json
```

Comment:
is see, that this is tested implicit but could you add some explicit tests for this unit please?

Reply:
done